ui/src/app/modules/components/tooltip/tooltip.component.scss

```diff
@@ -17,23 +17,83 @@
 
 @import 'colors';
 
+@media screen and (prefers-reduced-motion: reduce) {
```

Comment:
What all happened in this file? The diff view is kinda rough.

Reply:
Pretty sure it was just stylelint reordering things

Reply:
Okay gotcha. Thing I was worried about was I only saw the `prefers-reduced-motion` stuff in the add column so I thought maybe there was actual work mixed in with it.

Reply:
If so it was before I got to it

Reply:
Hmmm... @mschu124 any ideas?

Reply:
I think it's an ADA thing if I recall. I added it to some react components that have a css transition.
https://github.com/FordLabs/retroquest/blob/c29936e1893573654137ce3316ee543bc12efdc6/ui/src/react/components/action-item/ActionItem.scss#L93-L95

It doesn't show up in the Angular code until this commit, by @aclift19Ford
https://github.com/FordLabs/retroquest/commit/0b7ab3af53965018826862dbd9c7225d79c09d01

Reply:
Wild. I don't remember putting it there. Let me see if I can figure out what happened.

Reply:
It's gotta be from stylelint's auto fix. I didn't manually add those and I'm seeing it a lot other places.

Reply:
Yeah, this was def not a manual add. My bad for not reviewing style lints auto fixes better.

<img width="693" alt="Screen Shot 2022-01-19 at 5 05 04 PM" src="https://user-images.githubusercontent.com/74197928/150221656-f8fbee1c-33fa-4a11-8703-bfe4d87f7be9.png">

Reply:
Will revert tomorrow

Reply:
@LowBudgetMan Update has been added